tests/e2e/parallel/main_test.go

```diff
@@ -2431,6 +2570,9 @@ func TestManualRulesTailoredProfile(t *testing.T) {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      suiteName,
 			Namespace: f.OperatorNamespace,
+			Labels: map[string]string{
+				compv1alpha1.OutdatedReferenceValidationDisable: "true",
```

Comment:
How come we need to disable validation here?

Reply:
that's a good catch, I will remove it. It was left from the last approach, where we manually mark migrated rules, now we will not have these issues.

Reply:
This change is marked as as outdated, but still seems to be there at:
https://github.com/ComplianceAsCode/compliance-operator/pull/401/files#diff-a5334e69e88b595448bc22c081212911d6b0f04c2e91752ac2ef1ef7cb68c9edR2664
Is is expected to be there?